ndcube/tests/test_ndcubesequence.py

```diff
@@ -134,6 +134,15 @@ def test_cube_like_shape(ndc, expected_shape):
     assert np.all(ndc.cube_like_shape == expected_shape)
 
 
+@pytest.mark.parametrize(("ndc", "expected_dimensions"),
+                         [
+                             ("ndcubesequence_4c_ln_lt_l_cax1", tuple(u.Quantity(d, unit=u.pix) for d in [2. , 12, 4])),
+                         ],
+                         indirect=("ndc",))
+def test_cube_like_dimensions(ndc, expected_dimensions):
+    assert np.all(ndc.cube_like_dimensions == expected_dimensions)
```

Comment:
Astropy provides a [function](https://docs.astropy.org/en/latest/api/astropy.tests.helper.assert_quantity_allclose.html) to check two quantities are equal or almost equal with numerical precision.  I think that should be used here.  So something like 
```suggestion
    assert all(assert_quantity_allclose(ndc_dim, exp_dim) for ndc_dim, exp_dim in zip(ndc.cube_like_dimensions, expected_dimensions))
```

Reply:
If you would like to accept this change, you can do so on GitHub by clicking Commit suggestion.  Or, if you wanted to accepted multiple accepted changes, you can click add to batch (when looking at the "files" tab of the PR), and then commit all the suggested changes at once.

Just be sure to do `git pull me cubeLikeDimIssue` on your local machine afterwards to get this change on your own computer.